[2024-08-29T14:16:30.724+0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-29T14:16:30.738+0300] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_data_pipeline.load_data_to_bigquery scheduled__2024-08-28T00:00:00+00:00 [queued]>
[2024-08-29T14:16:30.743+0300] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_data_pipeline.load_data_to_bigquery scheduled__2024-08-28T00:00:00+00:00 [queued]>
[2024-08-29T14:16:30.743+0300] {taskinstance.py:2856} INFO - Starting attempt 4 of 4
[2024-08-29T14:16:30.753+0300] {taskinstance.py:2879} INFO - Executing <Task(GCSToBigQueryOperator): load_data_to_bigquery> on 2024-08-28 00:00:00+00:00
[2024-08-29T14:16:30.760+0300] {logging_mixin.py:190} WARNING - /home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=49754) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-29T14:16:30.761+0300] {standard_task_runner.py:72} INFO - Started process 49777 to run task
[2024-08-29T14:16:30.761+0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'reports_data_pipeline', 'load_data_to_bigquery', 'scheduled__2024-08-28T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/setup.py', '--cfg-path', '/tmp/tmpfxtbrxq5']
[2024-08-29T14:16:30.762+0300] {standard_task_runner.py:105} INFO - Job 19: Subtask load_data_to_bigquery
[2024-08-29T14:16:30.804+0300] {task_command.py:467} INFO - Running <TaskInstance: reports_data_pipeline.load_data_to_bigquery scheduled__2024-08-28T00:00:00+00:00 [running]> on host sa
[2024-08-29T14:16:30.877+0300] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='tc4a' AIRFLOW_CTX_DAG_ID='reports_data_pipeline' AIRFLOW_CTX_TASK_ID='load_data_to_bigquery' AIRFLOW_CTX_EXECUTION_DATE='2024-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-28T00:00:00+00:00'
[2024-08-29T14:16:30.878+0300] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-29T14:16:30.893+0300] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-08-29T14:16:30.895+0300] {base.py:84} INFO - Retrieving connection 'gcp_conn'
[2024-08-29T14:16:30.895+0300] {gcs_to_bigquery.py:381} INFO - Using existing BigQuery table for storing data...
[2024-08-29T14:16:30.927+0300] {gcs_to_bigquery.py:385} INFO - Executing: {'load': {'autodetect': True, 'createDisposition': 'CREATE_IF_NEEDED', 'destinationTable': {'projectId': 'visualization-app-404406', 'datasetId': 'tc4a', 'tableId': 'zoom'}, 'sourceFormat': 'CSV', 'sourceUris': ['gs://tc4a-backet/zoom/zoom_report_*.csv'], 'writeDisposition': 'WRITE_TRUNCATE', 'ignoreUnknownValues': False, 'skipLeadingRows': 1, 'fieldDelimiter': ',', 'quote': None, 'allowQuotedNewlines': False, 'encoding': 'UTF-8'}}
[2024-08-29T14:16:30.927+0300] {bigquery.py:1702} INFO - Inserting job airflow_reports_data_pipeline_load_data_to_bigquery_2024_08_28T00_00_00_00_00_7d4bf62200042f4b83ac95d0393e085c
[2024-08-29T14:16:33.367+0300] {taskinstance.py:3301} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py", line 446, in execute
    job.result(timeout=self.result_timeout, retry=self.result_retry)
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/google/cloud/bigquery/job/base.py", line 971, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/google/api_core/future/polling.py", line 261, in result
    raise self._exception
google.api_core.exceptions.NotFound: 404 Not found: Uris gs://tc4a-backet/zoom/zoom_report_*.csv; reason: notFound, message: Not found: Uris gs://tc4a-backet/zoom/zoom_report_*.csv
[2024-08-29T14:16:33.375+0300] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=reports_data_pipeline, task_id=load_data_to_bigquery, run_id=scheduled__2024-08-28T00:00:00+00:00, execution_date=20240828T000000, start_date=20240829T111630, end_date=20240829T111633
[2024-08-29T14:16:33.394+0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-29T14:16:33.395+0300] {standard_task_runner.py:124} ERROR - Failed to execute job 19 for task load_data_to_bigquery (404 Not found: Uris gs://tc4a-backet/zoom/zoom_report_*.csv; reason: notFound, message: Not found: Uris gs://tc4a-backet/zoom/zoom_report_*.csv; 49777)
Traceback (most recent call last):
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/models/taskinstance.py", line 2995, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/models/taskinstance.py", line 3149, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/models/taskinstance.py", line 3173, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py", line 446, in execute
    job.result(timeout=self.result_timeout, retry=self.result_retry)
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/google/cloud/bigquery/job/base.py", line 971, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sang/Documents/tc4a/airflow/env/lib64/python3.12/site-packages/google/api_core/future/polling.py", line 261, in result
    raise self._exception
google.api_core.exceptions.NotFound: 404 Not found: Uris gs://tc4a-backet/zoom/zoom_report_*.csv; reason: notFound, message: Not found: Uris gs://tc4a-backet/zoom/zoom_report_*.csv
[2024-08-29T14:16:33.423+0300] {local_task_job_runner.py:261} INFO - Task exited with return code 1
[2024-08-29T14:16:33.437+0300] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-29T14:16:33.441+0300] {local_task_job_runner.py:240} INFO - ::endgroup::
